README.md

```diff
@@ -8,11 +8,23 @@
 implementation 'com.github.appwise-labs:AndroidCore:<Latest-Version>'
 ```
 
+From v1.1.0 the library has grown to use multiple modules for specific use-cases. 
+The core can be added by using this dependency.
+```groovy
+implementation 'com.github.appwise-labs:AndroidCore:core:<Latest-Version>'
+```
+
+To use another module of this library just follow the folder structure
+```groovy
+implementation 'com.github.appwise-labs:AndroidCore:list:emptyRecyclerView:<Latest-Version>'
```

Comment:
Shouldn't we show a list of all the possible core submodules and a short explanation about what's in them?

Reply:
That's a nice idea, in my opinion, this could even be a separate ticket as the complete ReadMe structure could be reformatted. Possibly even with links to other 'ReadMe' files specific per submodule. If it's okay I'll create a new ticket for specifically that purpose!